ledger/src/tests.rs

```diff
@@ -206,6 +206,8 @@ fn test_insufficient_public_fees() {
     // Sample recipient.
     let recipient_private_key = PrivateKey::new(rng).unwrap();
     let recipient_address = Address::try_from(&recipient_private_key).unwrap();
+    let withdrawal_private_key = PrivateKey::<MainnetV0>::new(rng).unwrap();
```

Comment:
Most other tests use an intermediary `CurrentNetwork`, please adjust accordingly for ease of review